src/nrncvode/netcvode.cpp

```diff
@@ -83,21 +84,21 @@ extern ReceiveFunc* pnt_receive_init;
 extern short* pnt_receive_size;
 extern short* nrn_is_artificial_; // should be bool but not using that type in c
 extern short* nrn_artcell_qindex_;
-void net_send(void**, double*, Point_process*, double, double);
-void net_move(void**, Point_process*, double);
-void artcell_net_send(void**, double*, Point_process*, double, double);
-void artcell_net_move(void**, Point_process*, double);
+extern "C" void net_send(void**, double*, Point_process*, double, double);
+extern "C" void net_move(void**, Point_process*, double);
+//void artcell_net_send(void**, double*, Point_process*, double, double);
```

Comment:
how come those got commented out?

Reply:
extern "C" trial and error, will put them back